packages/http/src/validator/validators/utils.ts

```diff
@@ -7,8 +7,14 @@ import type { ErrorObject } from 'ajv';
 import { JSONSchema } from '../../';
 import * as AjvOAI from 'ajv-oai';
 
-const ajv = new AjvOAI({ allErrors: true, messages: true, schemaId: 'auto' });
-const ajvNoCoerce = new AjvOAI({ allErrors: true, messages: true, schemaId: 'auto', coerceTypes: false });
+const ajv = new AjvOAI({ allErrors: true, nullable: true, messages: true, schemaId: 'auto' });
```

Comment:
@chohmann 

I still keep receiving notifications — I hope you do not mind me chiming in.

The only reason we are using Ajv OAI is for the formats that it includes, not for the transformation capabilities. All the OpenAPI Specific schemas should be transformed by Http Spec, and that has been correctly (as far as I remember) happening until https://github.com/stoplightio/http-spec/pull/128 landed in.

The idea was not to let OAS-specific data to leak in Prism; Http Spec should be the piece of software handling that part, and that's where the bug should be fixed.

Additionally, I have received an email from NPM with an attempt to republish the fix; not too sure what happened, but the version currently published _might not_ be the one you intended to.

Reply:
@XVincentX your concerns are already being addressed in https://github.com/stoplightio/prism/pull/1783. 

>Additionally, I have received an email from NPM with an attempt to republish the fix; not too sure what happened, but the version currently published might not be the one you intended to.

🤔  Hmm that is strange. Well the binaries and source code attached to 4.1.3 tag all look correct, and the 4.1.3 version in NPM wasn't published until we published those, so I'm thinking it's correct.